inc/issue.class.php

```diff
@@ -211,13 +211,19 @@ public static function getSyncIssuesRequest() : AbstractQuery {
       ];
 
       // assistance requests having only one generated ticket (reuse query2)
-      $query3 = $query2;
-      // replace LEFT JOIN with INNER JOIN to exclude tickets not linked to a Formanswer object
-      $query3['INNER JOIN'][$itemTicketTable] = $query3['LEFT JOIN'][$itemTicketTable];
-      unset($query3['LEFT JOIN'][$itemTicketTable]);
-      // Only 1 relation to a Formanswer object
-      $query3['GROUPBY'] = ["$itemTicketTable.items_id"];
-      $query3['HAVING'] = new QueryExpression("COUNT(`$itemTicketTable`.`items_id`) = 1");
+      $query3 = [
+         'SELECT'     => $query2['SELECT'],
+         'FROM'       => $query2['FROM'],
+         'INNER JOIN' => [$itemTicketTable => $query2['LEFT JOIN'][$itemTicketTable]],
+         'LEFT JOIN'  => [
+            $query2['LEFT JOIN'][0], // This is the TABLE => [...] subquery
+            $ticketValidationTable => $query2['LEFT JOIN'][$ticketValidationTable],
+         ],
+         'WHERE'      => $query2['WHERE'],
+         'GROUPBY'    => ["$itemTicketTable.items_id"],
+         // Only 1 relation to a Formanswer object
+         'HAVING'     => new QueryExpression("COUNT(`$itemTicketTable`.`items_id`) = 1"),
+      ];
```

Comment:
this rewrite aims to convert tne INNER JOIN $itemTicketTable to LEFT JOIN $itemTicketTable without moving it on the bottom of the array representing the SQL. This is the 1st critical point.